src/PluginsInstaller.php

```diff
@@ -58,17 +54,18 @@ public static function possibly_install_activate_plugins() {
 				break;
 		}
 
-		self::cache_results( $install_result, $activate_result );
+		self::cache_results( $plugins, $install_result, $activate_result );
 		self::redirect_to_referer();
 	}
 
 	/**
 	 * Display the results of installation and activation on the page.
 	 *
-	 * @param array $install_result Result of installation.
-	 * @param array $activate_result Result of activation.
+	 * @param string $plugins Comma separated list of plugins.
+	 * @param array  $install_result Result of installation.
+	 * @param array  $activate_result Result of activation.
 	 */
-	public static function cache_results( $install_result, $activate_result ) {
+	public static function cache_results( $plugins, $install_result, $activate_result ) {
```

Comment:
An exception occurs on line 73 if the `plugins` parameter is empty (triggering `src/API/Plugins.php:224`), causing a error page, I don't know if that should be fixed in this PR though.